app/api/users.py

```
@@ -0,0 +1,54 @@
+        headers={"WWW-Authenticate": "Bearer"},
+    )
+    payload = user_service.decode_access_token(token)
+    if payload is None or "sub" not in payload:
+        raise credentials_exception
+    user_id = int(payload["sub"])
+    user = await user_service.get_user_by_email(db, payload.get("email"))
+    if user is None or user.id != user_id:
+        raise credentials_exception
+    return user
+
+@router.get("/me", response_model=UserResponse)
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)